parquet-hadoop/src/main/java/org/apache/parquet/ParquetReadOptions.java

```diff
@@ -185,6 +232,31 @@ public static class Builder {
     protected int maxAllocationSize = ALLOCATION_SIZE_DEFAULT;
     protected Map<String, String> properties = new HashMap<>();
     protected FileDecryptionProperties fileDecryptionProperties = null;
+    protected ParquetConfiguration conf;
+
+    public Builder() {
```

Comment:
Seems this constructor is not necessary.

Reply:
As this is public, removing the implicit no-args constructor _could_ potentially cause problems for users.

Reply:
I see the usage, that makes sense.